.github/workflows/build.yml

```diff
@@ -28,18 +28,15 @@ jobs:
       - uses: actions/cache@v2
         with:
           path: "**/dist/"
-          key: ${{ runner.os }}-tsc-${{ hashFiles('**/tsconfig.json') }}-${{ hashFiles('**/pnpm-lock.yaml') }}-${{ steps.get-date.outputs.date }}
-          restore-keys: |
-            ${{ runner.os }}-tsc-${{ hashFiles('**/tsconfig.json') }}-${{ hashFiles('**/pnpm-lock.yaml') }}
-            ${{ runner.os }}-tsc-${{ hashFiles('**/tsconfig.json') }}
+          key: ${{ runner.os }}-tsc-${{ hashFiles('packages/**/tsconfig.json') }}-${{ hashFiles('pnpm-lock.yaml') }}-${{ steps.get-date.outputs.date }}
+          restore-keys: ${{ runner.os }}-tsc-${{ hashFiles('packages/**/tsconfig.json') }}-${{ hashFiles('pnpm-lock.yaml') }}
```

Comment:
This change removes the ability to a fallback cache when the lock file has changed, which is tolerable in the tsc case. I'd prefer to revert this change.